bin/notifications.sh

```diff
@@ -195,7 +197,27 @@ function notification() {
 					log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "Email disabled and not creating file: ${_project_state_file}.mailed"
 					continue
 				else
-					sendEmail "${_project_state_file}" "${_project}" "${_run}" "${_phase}" "${_state}" "${_lfs_root_dir}" "${_controlFileBase}"
+					sendEmail='true'
+					IFS='/' read -r -a _traceSpecifications <<< "${_action}"
+					maxTime="${_traceSpecifications[1]:-0}" 
+					if [[ "${maxTime}" -ne '0' ]]
+					then
+						maxTimeMin=$((${maxTime}*60))
+						timeStampPipeline=$(find "${_project_state_file}" -mmin +"${maxTimeMin}")
```

Comment:
Confusing variable name. It is the time stamp of `${_project_state_file}`, which may be `pipeline.started`, but could be any other `phase.state`.